### PR TITLE
fix: simplify dynamic page

### PR DESCRIPTION
**Pull request by JKMarkowski**
## Related Issue
Closes https://github.com/SAP/fundamental-styles/issues/2128

## Description
There is refactor  of dynamic page - A lot of classes have been removed / updated. 
Examples have been changed to keep cozy buttons / cozy footers.
Placement for toolbars with buttons has been changed. For
- Desktop - it's aligned with title in single line
- Tablet / Mobile - first row is aligned with breadcrumb, second is aligned with a the tite

## BREAKING CHANGE:
- Removed all of size modifier classes with `--sm/--md/--lg/--xl`
- Removed `fd-dynamic-page__actions-container` - Additional container is not needed anymore
- `fd-dynamic-page__toolbar-actions` changed to `fd-dynamic-page__toolbar--actions`
- `fd-dynamic-page__content-toolbar` changed to `fd-dynamic-page__toolbar--content`
- `fd-tab` size modifier are not needed anymore 
- Placement of toolbars in DOM with buttons changed positions in tablet and desktop mode: 
  Now should be wrapped inside `fd-dynamic-page__breadcrumb-container` or `fd-dynamic-page__title-container`

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
- [x] hover state of the element follow design spec
- [x] focus state of the element follow design spec
- [x] active state of the element follow design spec
- [x] selected state of the element follow design spec
- [x] selected hover state of the element follow design spec
- [x] pressed state of the element follow design spec
- [x] Responsiveness rules - the component has modifier classes for all breakpoints
- [x] Includes Compact/Cosy/Tablet design
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [x] `fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [x] Storybook documentation has been created/updated
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

File: src/dynamic-page.scss [hunk 1]
```diff
@@ -458,38 +366,94 @@ $block: #{$fd-namespace}-dynamic-page;
...
 
-    &--md,
-    &--lg {
-      @include responsive-page($fd-dynamic-page-header-padding-top-bottom, 2rem);
+    .#{$block}__toolbar {
+      padding-left: 1rem;
+      margin-left: auto;
+
+      @include fd-rtl() {
+        margin-left: initial;
```
Comment (InnaAtanasova) [inline src/dynamic-page.scss:401]: `initial` does not work in IE11

File: src/dynamic-page.scss [hunk 2]
```diff
@@ -211,89 +214,37 @@ $block: #{$fd-namespace}-dynamic-page;
...
-    &--md {
-      flex-flow: row;
+    margin-left: auto;
 
-      .#{$block}__toolbar-container {
-        flex-flow: column wrap;
-      }
+    @include fd-rtl() {
+      margin-right: auto;
+      margin-left: initial;
```
Comment (InnaAtanasova) [inline src/dynamic-page.scss:225]: `initial` does not work in IE11

File: src/dynamic-page.scss [hunk 3]
```diff
@@ -458,38 +366,94 @@ $block: #{$fd-namespace}-dynamic-page;
...
-      @include responsive-page($fd-dynamic-page-header-padding-top-bottom, 2rem);
+    .#{$block}__toolbar {
+      padding-left: 1rem;
+      margin-left: auto;
+
+      @include fd-rtl() {
+        margin-left: initial;
+        margin-right: auto;
+        padding-left: 0;
+        padding-right: 1rem;
```
Comment (InnaAtanasova) [inline src/dynamic-page.scss:404]: this should be in a variable

File: src/dynamic-page.scss [hunk 4]
```diff
@@ -458,38 +366,94 @@ $block: #{$fd-namespace}-dynamic-page;
...
+      }
+
+      &--content {
+        margin-left: initial;
```
Comment (InnaAtanasova) [inline src/dynamic-page.scss:408]: same here for IE11

File: src/dynamic-page.scss [hunk 5]
```diff
@@ -458,38 +366,94 @@ $block: #{$fd-namespace}-dynamic-page;
...
+
+        @include fd-rtl() {
+          margin-right: initial;
```
Comment (InnaAtanasova) [inline src/dynamic-page.scss:411]: and here

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

**Comment by JKMarkowski** [comment]
@InnaAtanasova Thank you for review, you comments have been addressed

**Review by InnaAtanasova — APPROVED** [review]

**Review by droshev — APPROVED** [review]

**Pull request merged**